### PR TITLE
Add Slackware package

### LINKED ISSUE #9: FreeBSD packaging
### Role

-

### Suggestion

FreeBSD packaging

### Benefit

To install it on FreeBSD

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

Upstream added FreeBSD support in [3efcccf334e28d03605dd0597f53f900105cf04b](https://github.com/PolyMC/PolyMC/commit/3efcccf334e28d03605dd0597f53f900105cf04b). Currently we don't have builds for this. This issue is to track the progress of FreeBSD packaging

### PR DESCRIPTION
**Pull request by glowiak**

### CODE REVIEW

File: README.md
Comment (ghost) [inline README.md]: nit: command should be in `

**Review by ghost — DISMISSED** [review]

**Comment by d-513** [comment]
slackware should be as an h3 near other linux distro instructions
I think the link to prebuilt should be a static one. Not sure if you can replace the package when new version is released without another PR to PolyMC
google drive is also a werid way of hosting downloads

File: README.md
Comment (d-513) [inline README.md]:
```suggestion
- [AppBSD Image](https://drive.google.com/file/d/1QLYWGkLSjVb8-lhICivozZQDwYEWBack/view?usp=sharing) - a portable application, requires [AppBSD](https://codeberg.org/glowiak/appbsd/) to be installed.

- [Gzipped binaries](https://codeberg.org/attachments/7aa74368-1f55-49be-abd8-0b8258e63e38) - traditional way to distribute, unpack and run.
``` 

fixed formatting

File: README.md
Comment (d-513) [inline README.md]: what about using https://www.vectorlogo.zone/logos/freebsd/index.html ? this png scales pretty bad and has a background

```suggestion
## <img src="https://www.vectorlogo.zone/logos/freebsd/freebsd-icon.svg" height="20" /> FreeBSD
```

**Comment by ghost** [comment]
That's actually a really good point, why google drive?

Additionally, command-line instructions would be big if possible

**Comment by glowiak** [comment]
Ok, I'll make changes.
Why google drive? Why not? I have private files on yadi.sk, dropbox amount is too small to store all the images, and I want to avoid github everywhere I can

**Comment by glowiak** [comment]
Okay, now links are static, instructions added (though almost every Slackware or FreeBSD user would know how to do it), and moved Slack to the Linux section

**Comment by d-513** [comment]
I would change slackware to an h3 (three hashes), other than that LGTM!

**Comment by glowiak** [comment]
Oh ok, but anyway, Slackware is the only true Linux distro

**Comment by d-513** [comment]
Lol, anyways h3 just to match other distros because os family is h2 distros h3

**Comment by glowiak** [comment]
window$ belongs to DO$ family, macos is a proprietary copy of FreeBSD

**Comment by glowiak** [comment]
Changed Slackware to h3

**Review by d-513 — APPROVED** [review]

**Comment by glowiak** [comment]
So, will you merge the pull request?

**Comment by d-513** [comment]
I cant im just a random person. the maintainers are in a different timezone

**Comment by glowiak** [comment]
I'm compiling PMC to OpenBSD

**Comment by glowiak** [comment]
added openbsd

**Comment by ghost** [comment]
Instructions to build packages would be nice. I'd like to mirror these (or atleast the BSD packages) to packages.polymc.org.

**Comment by glowiak** [comment]
1. Of course, you can copy/mirror these packages.
2. They were built from sources as the BUILD.md file intended, just on OpenBSD changes launcher/tools/MCEditTool.cpp and launcher/UpdateController.cpp to include OBSD. I can make a separate PR to not have to change these files

**Comment by glowiak** [comment]
Though mirroring also the Slackware package would be nice

**Comment by glowiak** [comment]
Can anyone merge this?

**Comment by ghost** [comment]
@LennyMcLennington @txtsd

File: README.md
Comment (glowiak) [reply README.md]: Added

**Review by txtsd — APPROVED** [review]
LGTM!

**Comment by glowiak** [comment]
Could you merge my PRs?

**Comment by ghost** [comment]
fix conflicts and then @LennyMcLennington

**Comment by d-513** [comment]
@glowiak please fix conflicts so that we can merge

**Review by Scrumplex — APPROVED** [review]

**Comment by LennyMcLennington** [comment]
@glowiak btw what's the best way to contact you about the packages if we need to

**Comment by glowiak** [comment]
Mail glowiak1111@yandex.com, could you finally merge this pr?

**Comment by d-513** [comment]
@glowiak as we said you need to resolve conflicts in README.md before it is merged.

**Comment by glowiak** [comment]
But it's marked as resolved. What do I need to change?

**Comment by d-513** [comment]
> But it's marked as resolved. What do I need to change?

Git conflicts. Develop has changes that conflict with your changes. You need to resolve them via GitHub web using "Resolve Conflicts" button or git cli. Then we can merge it

image

**Review by ghost — DISMISSED** [review]
instructions for BSD manual builds would be cool, that way I can put some on Jenkins

**Comment by d-513** [comment]
> instructions for BSD manual builds would be cool, that way I can put some on Jenkins

they are in another PR

**Review by timoreo22 — APPROVED** [review]

**Comment by glowiak** [comment]
Could you finally merge this pr?

**Comment by ghost** [comment]
> > instructions for BSD manual builds would be cool, that way I can put some on Jenkins
> 
> they are in another PR

I mean an actual BSD binary package. I don't know if it's just "unpack a generic tarball and run it" or if there's some special file BSD uses. If not then instructions on creating an AppBSD would be cool

**Pull request merged**
